LFS-822: Reimport cardiac_rehab forms

acrowthe opened the pull request:
- Adds new CPET form and required import support
- Minor adjustments to several other forms
  - Additional computations or questions
  - Changed question types or units
- Converted bl_med2 questions to a recurrent sections

acrowthe committed:
LFS-822: Reimport cardiac_rehab forms
- Adds new CPET form and required import support
- Minor adjustments to several other forms
  - Additional computations or questions
  - Changed question types or units
- Converted bl_med2 questions to a recurrent sections

acrowthe commented:
As this change adds a recurrent section to a questionnaire but does not update the import script with this functionality, [LFS-827](https://phenotips.atlassian.net/jira/software/c/projects/LFS/issues/LFS-827) has been created.

sashaandjic commented:
A very small complaint: as we split the Events form into Clinical Events and Medical change, it would be a good idea to have these 2 new forms next 2 each other in the selection dialog:
![image](https://user-images.githubusercontent.com/23243244/103866959-962f5b80-5094-11eb-875a-57c761c34678.png)
![image](https://user-images.githubusercontent.com/23243244/103866927-8e6fb700-5094-11eb-96d3-76ada06788a9.png)

sashaandjic commented:
We should remove this field from Clinical Event form.
![image](https://user-images.githubusercontent.com/23243244/103867594-9b40da80-5095-11eb-98c6-b784cef6eacf.png)

sashaandjic commented:
All of these changes are done:
![image](https://user-images.githubusercontent.com/23243244/103869133-ea880a80-5097-11eb-9753-8e5c4f574e5b.png)

acrowthe committed:
LFS-822: Reimport cardiac_rehab forms
- Rename clinical events and medication change forms to be alphabetically adjacent
- Remove type of event question and dependant conditionals in event forms

acrowthe committed:
LFS-822: Reimport cardiac_rehab forms
Fix maxPerSubject on clinical event, medication change and CPET forms

acrowthe committed:
LFS-822: Reimport cardiac_rehab forms
Adjust baseline forms
- Rename Baseline Health Information: `Medications/Cardio...` to
  `Baseline Medications/Cardio...`
- Move a few instances of `check all that apply` from question title
  to description

sashaandjic reviewed (approved):
tested - ALL GOOD!

k-pace commented:
Reason for pausing conditional doesn't seem to be working
![image](https://user-images.githubusercontent.com/33154316/103951806-55275d80-510d-11eb-96c0-6a5e88292403.png)

acrowthe committed:
LFS-822: Reimport cardiac_rehab forms
Reimport 6MWT form due to added conditionals
Reimport Baseling Medications form to adjust wording

k-pace commented:
This looks good to me - bug seems to have been fixed with the conditional questions.
Recurrent section for medications looks great!
Styling changes to move things contained in parentheses looks great - good call @sashaandjic !
